### PR TITLE
fix: display stderr on pact publish process failure

### PR DESCRIPTION
**Pull request by WaylandAce**
Display stderr on pact publish process failure

Unify behavior with https://github.com/pact-foundation/pact-php/blob/master/src/PhpPact/Standalone/ProviderVerifier/VerifierProcess.php#L65

### CODE REVIEW

**Review by Big-Shark — APPROVED** [review]

**Comment by cfmack** [comment]
Simple enough (and thank you).   Looks like we are failing a Lint test @WaylandAce .   Can you help?  https://github.com/pact-foundation/pact-php/actions/runs/4086790206/jobs/7182694903

File: src/PhpPact/Standalone/Broker/Broker.php [hunk 1]
```diff
@@ -245,13 +245,18 @@ public function publish(): void
...
 
-        $this->logger->debug($runner->getOutput());
+            $this->logger->info('out > ' . $runner->getOutput());
+            $this->logger->error('err > ' . $runner->getStderr());
+        } catch (\Exception $e) {
+            $this->logger->info('out > ' . $runner->getOutput());
+            $this->logger->error('err > ' . $runner->getStderr());
+
+            throw $e;
+        }
```
Comment (tienvx) [inline src/PhpPact/Standalone/Broker/Broker.php:259]:
```suggestion
try {
            $runner->runBlocking();
} finally {
            $this->logger->info('out > ' . $runner->getOutput());
            $this->logger->error('err > ' . $runner->getStderr());
}
```
I think this code is shorter and work the same.

Comment (WaylandAce) [reply src/PhpPact/Standalone/Broker/Broker.php:259]: But we need re-throw exception also, not just output

Comment (tienvx) [reply src/PhpPact/Standalone/Broker/Broker.php:259]: I tested with this code and I think the exception will be re-throwed.

```php
<?php

function throwException() {
    throw new Exception('test');
}

try {
    print("Outer Try\n");
    try {
        print("Try\n");
        throwException();
    } finally {
        print("Finally\n");
    }
} catch (\Throwable $th) {
    printf("Outer Catch: %s\n", $th->getMessage());
} finally {
    print("Outer Finally\n");
}

```

Output:

```php
Outer Try
Try
Finally
Outer Catch: test
Outer Finally
```

**Review by tienvx — APPROVED** [review]

**Pull request merged**